crawl-ref/source/webserver/game_data/static/action_panel.js

```diff
@@ -386,13 +452,10 @@ function ($, comm, client, cr, enums, options, player, icons, gui, util,
                               _horizontal() ? panel_length : cell_width,
                               _horizontal() ? cell_height : panel_length);
 
-        // XX This should definitely be a different/custom icon
-        renderer.draw_gui(gui.PROMPT_NO, 0, 0, adjusted_scale / 100);
-        if (selected == 0)
-        {
-            renderer.draw_icon(icons.CURSOR3, 0, 0, undefined, undefined,
-                adjusted_scale / 100);
-        }
+        // XX The "X" should definitely be a different/custom icon
+        draw_action(gui, gui.PROMPT_NO, 0, adjusted_scale, selected == 0);
+        draw_action(gui, gui.STARTUP_STONESOUP, cell_length, adjusted_scale,
```

Comment:
it's worth adding a TODO note here, something like:
```
// TODO: select tile via something like c++ `tileidx_command`
```

(Currently no way to do this in js that I know of, probably not worth doing in this PR but worth flagging.)